Runtime/AvatarCreator/Scripts/UI/Elements/AssetSelectionElement.cs

```diff
@@ -21,11 +21,12 @@ public class AssetSelectionElement : SelectionElement
         [SerializeField] private int iconSize = 64;
 
         private PartnerAsset[] assets;
+        
         private AssetAPIRequests assetsRequests;
+        private AssetAPIRequests AssetsRequests => assetsRequests ??= new AssetAPIRequests(CoreSettingsHandler.CoreSettings.AppId);
```

Comment:
Why not just use this ? 
private AssetAPIRequests assetsRequests = new AssetAPIRequests(CoreSettingsHandler.CoreSettings.AppId); 
Is the extra "AssetRequests" property needed? It's also private so not sure it makes sense to keep it there?